### PR TITLE
Add rules announcement email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add an announcement emails for the rules for SR2019. This should be sent once https://github.com/srobo/website/pull/111 has shipped, and obviously once the rules are uploaded too.

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
This is just to announce the rules, rather than a "we had a good kickstart" email as https://github.com/srobo/website/pull/111 is, so I think it's fine not talk about everything

**Comment by prophile** [comment]
We're probably "publishing" these rather than "releasing" them: people already know what the rules are.

File: SR2019/2018-11-13-rules-announce.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Hi all,
+
+It's with great pleasure we're publishing the rulebook for Student Robotics 2019. We announced the game at [kickstart](https://www.studentrobotics.org/2018-11-12-sr2019-rules-released/), and gave you an insight into the rules, but the rulebook formalises and documents them for all to see.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries.
+
+[Download the rules](https://studentrobotics.org/docs/rules/)
+
+If you have any questions about the rules, please [post on our forums](https://www.studentrobotics.org/forum/) (which will be live very soon!).
```
Comment (PeterJCLaw) [inline SR2019/2018-11-13-rules-announce.md]: The forums will be live before we ship things.

File: SR2019/2018-11-13-rules-announce.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+to: Student Robotics 2019
+subject: Rules for SR2019 released
```
Comment (PeterJCLaw) [inline SR2019/2018-11-13-rules-announce.md]:
```suggestion
subject: Rules for SR2019 published
```

File: SR2019/2018-11-13-rules-announce.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+to: Student Robotics 2019
+subject: Rules for SR2019 published
+---
+
+Hi all,
+
+It's with great pleasure we're publishing the rulebook for Student Robotics 2019. We announced the game at [kickstart](https://www.studentrobotics.org/2018-11-12-sr2019-rules-released/), and gave you an insight into the rules, but the rulebook formalises and documents them for all to see.
```
Comment (PeterJCLaw) [inline SR2019/2018-11-13-rules-announce.md]: The url to the post has changed -- I think it should now be:
```suggestion
It's with great pleasure we're publishing the rulebook for Student Robotics 2019. We announced the game at [kickstart](https://studentrobotics.org/news/2018-11-13-sr2019-rules-published/), and gave you an insight into the rules, but the rulebook formalises and documents them for all to see.
```

File: SR2019/2018-11-13-rules-announce.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries.
+
+[Download the rules](https://studentrobotics.org/docs/rules/)
+
+If you have any questions about the rules, please [post on our forums](https://www.studentrobotics.org/forum/).
```
Comment (PeterJCLaw) [inline SR2019/2018-11-13-rules-announce.md]: Minor: our website redirects to remove `www.`, so we can save visitors a redirect by doing that here too:
```suggestion
If you have any questions about the rules, please [post on our forums](https://studentrobotics.org/forum/).
```

**Comment by antoinepetty** [comment]
URL needs changing again to https://studentrobotics.org/news/2018-11-14-sr2019-rules-published/

File: SR2019/2018-11-13-rules-announce.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+subject: Rules for SR2019 published
+---
+
+Hi all,
+
+It's with great pleasure we're publishing the rulebook for Student Robotics 2019. We announced the game at [kickstart](https://studentrobotics.org/news/2018-11-14-sr2019-rules-published/), and gave you an insight into the rules, but the rulebook formalises and documents them for all to see. This rulebook has some notable changes compared to what was announced at kickstart:
+
+- The height of the volcano is now 75mm
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries.
```
Comment (PeterJCLaw) [inline SR2019/2018-11-13-rules-announce.md]: Minor: maybe also steal Antoine's framing for this too:
```suggestion
Our game this year, Caldera, challenges teams to control scoring zones with tokens while navigating an arena with raised platforms.
```

**Pull request merged**